File: packages/reflex-shared-types/src/config/prediction-settings.ts
export interface PredictionSettings {
  useVelocityPrediction: boolean;
  numFramesForPrediction: number;
  secondDerivationMagnitude: number;
  useSecondDerivation: boolean;
  filterPointsByVelocity: boolean;
  velocityFilterThreshold: number;
}

File: tools/ReFlex.TrackingServer/Data/Config/PredictionSettings.cs
namespace TrackingServer.Data.Config;

public class PredictionSettings
{
    public bool UseVelocityPrediction { get; set; } = true;

    public int NumFramesForPrediction { get; set; } = 2;

    public bool UseSecondDerivation { get; set; } = false;

    public float SecondDerivationMagnitude { get; set; } = 0.5f;

    public bool FilterPointsByVelocity { get; set; } = true;

    public float VelocityFilterThreshold { get; set; } = 2.5f;

    public PredictionSettings()
    {
    }


}